### PR TITLE
Change the alignment of visit log icons and modify Add new color to yellow

### PR DESCRIPTION
**Pull request by SindhujaReddy-VT**
Change the alignment of visit log icons and modify Add new color to yellow

### CODE REVIEW

**Comment by phillipsk** [comment]
The icons on the bottom navigation bar need to be updated with the yellow accent color as well

Screenshot_20240813_004250

File: app/src/main/res/layout/content_main.xml [hunk 1]
```diff
@@ -34,5 +34,6 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="parent"
-        app:menu="@menu/activity_main_drawer" />
+        app:menu="@menu/activity_main_drawer"
+        app:itemBackground="@animator/menu_item_background_selector"/>
```
Comment (phillipsk) [inline app/src/main/res/layout/content_main.xml:38]: Does this match iOS? or did you have another reason for adding this background rectangle?

Screenshot_20240817_202714

**Review by phillipsk — COMMENTED** [review]
Comment added on background selector

File: app/src/main/res/layout/content_main.xml (hunk 1 above)
Comment (SindhujaReddy-VT) [reply app/src/main/res/layout/content_main.xml:38]: The reason for using the background rectangle is that we already have it in Figma.

**Review by phillipsk — APPROVED** [review]

File: app/src/main/res/layout/content_main.xml (hunk 1 above)
Comment (phillipsk) [reply app/src/main/res/layout/content_main.xml:38]: The figma has many variations and I did not see this there, do you have the link to the design?

Comment (SindhujaReddy-VT) [reply app/src/main/res/layout/content_main.xml:38]: https://www.figma.com/design/V5mIYvVRm7dAfZYesOWLBX/Street-Care-App-Design-(2023-updated)?node-id=5453-24924&t=mua9P7cJDj9RAeGc-1

This link has all the figma designs

**Pull request merged**